### DISCUSSION TITLE
トラックボールの操作性について

### DISCUSSION THREAD

**Discussion by sndmtk** [Yowkees/keyball#186] [General]
トラックボールで円を描くような操作をすると少しずつ位置がズレていくようで（時計回りに円を描くとマウスは上に、反時計回りに円を描くとマウスは下へ）、元々の症状なのかセンサーの取り付けが原因なのか気になっています。センサーの光軸に対してマウスの移動量を調整するようなことで解決したりするのでしょうか。

https://gyazo.com/716338fe584957938d4a086ad9126e58

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

おそらくセンサー取り付け位置等は関係なく、ボールのどこに指を置いているかだと思います。
トラックボールはセンサーの真裏に指を置いたとき、もっとも動作が安定します。

私も今、Keyballで円を描いてみていますが、同じ位置をくるくると動かすことができます。

マウスの移動量を校正（調整）するためにはKeyballのソースコードを編集していただく必要があります。

ご検討よろしくお願いいたします。

**Reply by sndmtk** [reply]
ご回答ありがとうございます。

長らくMX Ergoを使っていた癖みたいなのも影響ありそうですね😅
